src/sdk.ts

```diff
@@ -56,4 +56,6 @@ export const getJwtRoles = wrapInTry(
 		sdkInstance?.getJwtRoles(token, tenant)
 );
 
+export const refreshToken = (token?: string) => sdkInstance?.refresh(token);
```

Comment:
```suggestion
export const refreshToken = (token = getRefreshToken()) => sdkInstance?.refresh(token);
```

Reply:
it is only the default value, it still allow empty token

 - it will work the same if jwt are on cookies (because `getRefreshToken` will return empty string)
 - but it will just be simpler if jwt are not on cookies, and in this case, descoper can call `refreshToken()` without argument)